File: backend/src/main/java/de/bund/digitalservice/ris/adm_vwv/application/Fundstelle.java
package de.bund.digitalservice.ris.adm_vwv.application;

import java.util.List;

/**
 * Fundstelle business object
 *
 * @param id          The id of the Fundstelle
 * @param zitatstelle The Zitatstelle of the Fundstelle
 * @param periodika   The list of Periodika of this Fundstelle
 */
public record Fundstelle(String id, String zitatstelle, List<Periodikum> periodika) {}

File: frontend/src/testing/fixtures/fundstelle.ts
import type { Fundstelle } from '@/domain/fundstelle'
import { bundesanzeigerFixture } from './periodikum'

export const fundstelleFixture: Fundstelle = {
  id: 'fundstelleTestId',
  zitatstelle: '1973, 608',
  periodikum: bundesanzeigerFixture,
}
